### DISCUSSION TITLE
Jest unable to find module

### DISCUSSION THREAD

**Discussion by ahamelers** [Milkdown/milkdown#1160] [Q&A]
I'm using milkdown in a react project. I have a Jest error:

```
Test suite failed to run

    Cannot find module '@milkdown/prose/model' from 'node_modules/@milkdown/core/lib/index.es.js'

    Require stack:
      node_modules/@milkdown/core/lib/index.es.js
```

I've got the latest version of Jest, using babel-jest with babel preset-react. Does anyone have any ideas to resolve this?

**Comment by dark-panda** [comment]
We're also seeing this issue with jest. We're not using React, but we're seeing the same issue.

**Comment by NadineJo** [comment]
We have the same issue with Jest in an Angular project. Has anyone found a solution for this?

**Comment by AlucardleVash** [comment]
Same issue here :) 
Does anyone find a solution ?
